Cargo.toml

```diff
@@ -22,9 +22,11 @@ A crate for interacting with the Alpaca API.
 include = ["src/**/*", "LICENSE", "README.*", "CHANGELOG.*"]
 
 [features]
-default = ["gzip"]
+default = ["gzip", "native-tls"]
 gzip = ["async-compression/futures-io", "async-compression/gzip"]
-vendored-openssl = ["hyper-tls/vendored"]
+vendored-openssl = ["hyper-tls?/vendored", "dep:hyper-tls"]
```

Comment:
I don't believe I understand the logic here. Why does `vendored-openssl` depend on `hyper-tls`?